src/elements/flip-card/flip-card/flip-card.visual.spec.ts

```diff
@@ -146,6 +167,37 @@ describe(`sbb-flip-card`, () => {
             await waitForImageReady(setup.snapshotElement.querySelector('sbb-image')!);
           }),
         );
+
+        it(
+          `long content`,
```

Comment:
Title seems not to fit the test? But was already before this PR not matching :)

Reply:
I don't know if the original intention was to test the flipped card with long content but, as it is now, it's not very helpful. 
Anyway, I wanted to rearrange other things but stopped since I needed to test for regressions first

Reply:
the idea was to do it in another pr where I only rearrange tests (to be sure I don't introduce any regression)